datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -4239,6 +4244,11 @@ type CorpUserHomePageSettings {
   The default page template for the User.
   """
   pageTemplate: DataHubPageTemplate
+
+  """
+  List of urns of the announcements dismissed by the User.
+  """
+  dismissedAnnouncements: [String]
```

Comment:
if we don't want to return the whole announcement entity (which sounds like we don't need to right now) let's maybe call this `dismissedAnnounementUrns` since that's clearer, otherwise it sounds like we should be able to check out an announcement entity in this list